feathr_project/feathr/definition/sink.py

```diff
@@ -333,3 +333,25 @@ def get_required_properties(self) -> List[str]:
         if self.auth:
             return [self.name.upper() + "_USER", self.name.upper() + "_PASSWORD"]
         return []
+
+class AerospikeSink(GenericSink):
+    def __init__(self,name:str,seedhost:str,port:str,namespace:str,setname:str):
```

Comment:
Does `port` have a default value? And should this parameter be a number instead of str? We can convert it into str on line 341.

Reply:
Changed to int, but did not set default value, since port 3000 can be redirected sometime...